### DISCUSSION TITLE
好像触发了防盗链机制 怎么解决

### DISCUSSION THREAD

**Discussion by zhengpan123** [houtianze/bypy#662] [Q&A]
![图片1](https://github.com/houtianze/bypy/assets/150119619/9a2d654a-914a-426a-bcad-06575722480c)

**Comment by zhengpan123** [comment]
前几天用的都挺正常的，只是下载速度只有几十kb/s，后来还开了个大会员，变成了几m/s。下了几个文件之后，过两天就这样，下不了了

**Comment by bookong** [comment]
同问，我的也是
